Buscorepuestos/buscorepuestos-web
# Feature/br 478 product cart info

## What type of PR is this? (check all applicable)


## Description
Se creo un nuevo componente para el preview del producto dentro del carrito de compra.


## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-478/product-cart-info

## QA Instructions, Screenshots, Recordings

Ejecute la rama y acceda a la url [http://localhost:3000/shopping-cart](url)
Verifique que el diseño corresponda y que sea responsive.
Ejecute los test y verifique que pasaron y que el coverage se mantiene al 100% 

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
